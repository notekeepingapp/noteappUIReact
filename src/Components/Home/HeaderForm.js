import React from "react";

const HeaderForm=() =>{
    return(
        <div>
            <div className={"container-fluid h-100 text-dark"}>
                <nav className="navbar">
                    <span className="mb-0 h1 navbar-text">Note Keeping app</span>
                </nav>
            </div>
            <div className="container">
                <div className="jumbotron row justify-content-center mt-0 p-0">
                    <h1>Your Notes</h1>
                </div>
            </div>
        </div>
    )
};
export default HeaderForm;