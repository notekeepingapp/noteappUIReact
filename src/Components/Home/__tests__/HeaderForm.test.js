import React from 'react';
import { shallow } from 'enzyme';
import HeaderForm from '../HeaderForm';

describe('HeaderForm rendering', () => {
  it('should match HeaderForm snapshot', () => {
    const component = shallow(<HeaderForm />);
    expect(component).toMatchSnapshot();
  });
});
