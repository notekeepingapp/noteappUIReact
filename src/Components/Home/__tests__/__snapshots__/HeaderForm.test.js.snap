// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderForm rendering should match HeaderForm snapshot 1`] = `ShallowWrapper {}`;
